projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 # HEAD (unreleased)
 
+- Fix: re-infer `json-editor` node types after model change
+
+## 31.0.1 (2020-11-03)
+
 - Fix: Change \* imports to a namespaces compatible version
 - Enhancement: Add `[withMargin]` option to `ngx-input`
```

Comment:
These "enhancements" (except for the imports fix) aren't released under 31.0.1 yet. Can we make a slight change to this?

Reply:
@nartc good catch. Let me know if it looks correct now.